### PR TITLE
4.4.0 template for vpc block csi driver

### PR DESCRIPTION
**Pull request by GunaKKIBM**
TODO
1. Testing using this template
2. Update images of vpc node label updater and csi driver in the deployment

### CODE REVIEW

File: config-templates/ibm/ibm-vpc-block-csi-driver/4.4.0/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,658 @@
+apiVersion: v1
+kind: List
+metadata:
+  name: ibm-vpc-block-csi-driver
+  namespace: kube-system
+  annotations:
+    version: 4.4.0
+    revision: 1
```
Comment (prankulmahajan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/4.4.0/deployment.yaml:8]: @mssachan Do we mention `revision:1` in the new template version?

File: config-templates/ibm/ibm-vpc-block-csi-driver/4.4.0/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,658 @@
+apiVersion: v1
+kind: List
+metadata:
+  name: ibm-vpc-block-csi-driver
+  namespace: kube-system
+  annotations:
+    version: 4.4.0
```
Comment (prankulmahajan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/4.4.0/deployment.yaml:7]: @GunaKKIBM We can rename `4.4.0` to just `4.4` everywhere to be in sync with the naming convention we have in managed environment.

File: config-templates/ibm/ibm-vpc-block-csi-driver/4.4.0/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,658 @@
...
+                - name: plugin-dir
+                  mountPath: /csi
+                - name: registration-dir
+                  mountPath: /registration
+            - name: iks-vpc-block-node-driver
+              securityContext:
+                runAsNonRoot: false
+                runAsUser: 0
+                privileged: true
+              image: icr.io/ibm/ibm-vpc-block-csi-driver:v4.3.5
```
Comment (prankulmahajan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/4.4.0/deployment.yaml:364]: We need to update this for latest version of 4.4

File: config-templates/ibm/ibm-vpc-block-csi-driver/4.4.0/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,658 @@
...
+                  memory: 40Mi
+                requests:
+                  cpu: 5m
+                  memory: 10Mi
+              volumeMounts:
+                - name: socket-dir
+                  mountPath: /csi
+            - name: iks-vpc-block-driver
+              imagePullPolicy: Always
+              image: icr.io/ibm/ibm-vpc-block-csi-driver:v4.3.5
```
Comment (prankulmahajan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/4.4.0/deployment.yaml:592]: Same here

File: config-templates/ibm/ibm-vpc-block-csi-driver/4.4.0/custom-parameters.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+[
+  {
+    "description": "The IAM endpoint. For example 'https://iam.cloud.ibm.com'.",
+    "displayname": "IAM endpoint",
+    "name": "g2_token_exchange_endpoint_url",
+    "required": "true",
+    "category": "config",
+    "mutable": "false"
+  }, {
+    "description": "The VPC regional endpoint of your VPC cluster in the format https://<region>.iaas.cloud.ibm.com. Example: https://eu-de.iaas.cloud.ibm.com. For more information, see https://cloud.ibm.com/docs/vpc?topic=vpc-service-endpoints-for-vpc",
```
Comment (prankulmahajan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/4.4.0/custom-parameters.json:11]: "The VPC regional endpoint of your VPC cluster in the format `https://region.iaas.cloud.ibm.com`. Example: https://eu-de.iaas.cloud.ibm.com. For more information, see https://ibm.biz/vpc-endpoints"
Ref https://github.com/IBM/ibm-satellite-storage/pull/204/files

**Review by prankulmahajan — CHANGES_REQUESTED** [review]
For naming convention we can confirm with @mssachan and @arahamad

**Comment by GunaKKIBM** [comment]
Closing this, as 5.0 is released

**Pull request closed without merging**